lib/Github/Api/CurrentUser.php

```diff
@@ -104,7 +104,6 @@ public function teams()
      * @param  string $type      role in the repository
      * @param  string $sort      sort by
      * @param  string $direction direction of sort, ask or desc
-
```

Comment:
The other's are blank lines of comment. This is just a dodgie blank line with nothing on it at all.